lib/resources/r4/coverage_encounter_patch.yaml

```diff
@@ -0,0 +1,163 @@
+    description: The alphanumeric string value associated with the insurer issued label.
+    example: |
+      {
+        "op": "replace",
+        "path": "/class/1/value",
+        "value": "729825"
+      }
+    note: >
+      <ul>
+        <li>This path should be used only to replace the policy group number.</li>
+        <li>When no value is provided the `value` for the `Coverage.class` element with `type` == `group` type will be unset.</li>
+      </ul>
+
+  - name: dependent
+    path: /dependent
+    operation: replace
+    cardinality: 0..1
+    type: string
+    url: https://hl7.org/fhir/coverage-definitions.html#Coverage.dependent
+    description: A unique identifier for a dependent under the coverage.
+    example: |
+      {
+        "op": "replace",
+        "path": "/dependent",
+        "value": "1626016"
+      }
+    note: >
+      <ul>
+        <li>When no value is provided `Coverage.dependent` will be unset.</li>
+      </ul>
+
+  - name: period
+    path: /period
+    operation: replace
+    cardinality: 0..1
+    type: Period
+    url: https://hl7.org/fhir/coverage-definitions.html#Coverage.period
+    description: Time period during which the coverage is in force. A missing start date indicates the start date isn't known, a missing end date means the coverage is continuing to be in force.
+    example: |
+      {
+        "op": "replace",
+        "path": "/period",
+        "value": {
+          "start": "2014-09-05T14:30:12.000Z",
+          "end": "2015-09-05T14:30:12.000Z"
+        }
+      }
+    note: >
+      <ul>
+        <li>When no `start` value is provided `Coverage.period.start` will be unset.</li>
+        <li>When no `end` value is provided `Coverage.period.end` will be unset.</li>
+        <li>When `start` or `end` values are provided, time components must be provided.</li>
+      </ul>
+
+  - name: relationship
+    path: /relationship
+    operation: replace
+    cardinality: 0..1
+    type: CodeableConcept
+    url: https://hl7.org/fhir/coverage-definitions.html#Coverage.relationship
+    description: The relationship of beneficiary (patient) to the subscriber.
+    example: |
+      {
+        "op": "replace",
+        "path": "/relationship",
+        "value": {
+          "coding": [
+            {
+              "system": "https://terminology.hl7.org/CodeSystem/subscriber-relationship",
+              "code": "parent"
+            }
+          ]
+        }
+      }
+    note: >
+      <ul>
+        <li>When no value is provided `Coverage.relationship` will be unset.</li>
+      </ul>
+
+  - name: subscriberId
+    path: /subscriberId
+    operation: replace
+    cardinality: 0..1
+    type: string
+    url: https://hl7.org/fhir/coverage-definitions.html#Coverage.subscriberId
+    description: The insurer assigned ID for the Subscriber.
+    example: |
+      {
+        "op": "replace",
+        "path": "/subscriberId",
+        "value": "1626016"
```

Comment:
can we change this value so that it doesn't match the id on the reference?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/342/commits/b31e12c97d001c4062999cecf9fa9db4b84409e8#diff-809caa732a11bff3f7f981f307721b86R138